uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/components/fields-templates/leave-type-wizard-field-default_entitlement.html

```diff
@@ -4,4 +4,8 @@
     <ng-include
       src="form.componentsPath + '/leave-type-wizard-field-error.html'"></ng-include>
   </div>
+  <div class="col-md-3 control-label" ng-switch="form.fieldsIndexed.calculation_unit.value">
+    <span ng-switch-when="1">days</span>
```

Comment:
<img width="703" alt="old_view" src="https://user-images.githubusercontent.com/1507645/47849689-e2a2f080-ddd2-11e8-98e3-fab2495508be.png">

Have a look at the above ☝️ and the mock up below 👇 

![mockup](https://user-images.githubusercontent.com/1507645/47849793-2990e600-ddd3-11e8-91ea-02a3ddc64eda.png)

Guess we can confirm from Shubam or @mi-kon. This will reduce confusion on the part of the tester. 🤷‍♂️

Reply:
I think the mockup tells us "show either this or that". The previous story has passed testing and we have the same dynamic label for the other field. So I guess we are good here.

Reply:
![proof](https://user-images.githubusercontent.com/3973243/47850825-83db7800-ddce-11e8-8d49-d0412f7a5fad.gif)

We are good.